src/adapters/scene.js

```diff
@@ -160,22 +160,17 @@ Scene.prototype.initMapBox = function() {
 };
 
 Scene.prototype.clickOnMap = function(lngLat, clickedFeature) {
-  // Disable POI anywhere feature on mobile until we opt for an adapted UX
-  if (isMobileDevice() && !clickedFeature) {
```

Comment:
I think you removed too much stuff here during the rebase. We still want specific behavior on mobile when not in direction mode, when clicking anywhere on the map.